src/scss/components/_inputs.scss

```diff
@@ -144,6 +144,27 @@ $input-select-margin-right: 1;
   }
 }
 
+.tile-container {
+  @include at-media('tablet') {
+    display: flex;
+  }
+  &__image {
+    width: 1em;
```

Comment:
I think we'll want to use `rem` units here and throughout so that doesn't scale to the container text size.